client/question/services/FeedbackGeneratorService.js

```diff
@@ -53,6 +61,57 @@ tie.factory('FeedbackGeneratorService', [
     var consecutiveSameRuntimeErrorCounter = 0;
 
     /**
+     * Object containing variations on feedback text for different scenarios.
+     *
+     * $type {Object.<string, Array.<string>>}
+     */
+    var correctnessFeedbackText = {
```

Comment:
Pull out into constant in questions.js, rename to CORRECTNESS_FEEDBACK_TEXT. Upper-caps the interior keys as well (these could be equal to the status names I suggested above).

Makes it easier to change just the text in the future, if needed.

Reply:
Yeah, this was originally a constant, but I went back and forth and eventually made it a regular variable, thinking it could be updated. OK, I'll change it back to constant and move it to questions.s.